vtm/src/org/oscim/renderer/bucket/PolygonBucket.java

```diff
@@ -333,28 +333,27 @@ public static RenderBucket draw(RenderBucket buckets, GLViewport v,
                     /* project bbox of polygon to screen */
                     v.mvp.prj2D(pb.bbox, 0, box, 0, 4);
 
-                    int out = 0;
+                    int out = LineClipper.INSIDE;
                     for (int i = 0; i < 8; i += 2) {
                         int o = mScreenClip.outcode(box[i], box[i + 1]);
 
-                        if (o == 0) {
+                        if (o == LineClipper.INSIDE) {
                             /* at least one corner is inside */
-                            out = 0;
+                            out = LineClipper.INSIDE;
                             break;
                         }
                         out |= o;
                     }
                     /* Check if any polygon-bucket edge intersects the screen.
                      * Also check the very unlikely case where the view might
-                     * be
-                     * completly contained within box */
-                    if ((out != 0) && (out != 0xF)) {
+                     * be completly contained within box */
+                    if ((out != LineClipper.INSIDE) && (out != LineClipper.OUTSIDE)) {
                         mScreenClip.clipStart(box[6], box[7]);
-                        out = 0;
-                        for (int i = 0; i < 8 && out == 0; i += 2)
+                        out = LineClipper.INSIDE;
+                        for (int i = 0; i < 8 && out == LineClipper.INSIDE; i += 2)
```

Comment:
Why then `0` is inside while before was outside?

Reply:
You're absolutely right! Didn't notice the while.